rest-api-spec/src/main/resources/rest-api-spec/test/README.md

```diff
@@ -319,13 +319,13 @@ Stashed values can be used in property names, eg:
     - do:
         cluster.state: {}
 
-    - set: { master_node: master }
+    - set: { cluster_manager_node: master }
```

Comment:
Do we need to replace `master` (in value) as well ?

Reply:
Yes. Changed it